src/istio/authn/context.proto

```diff
@@ -55,18 +55,10 @@ message X509Payload {
   // The trust domain corresponds to the trust root of a system.
   // Refer to
   // [SPIFFE-ID](https://github.com/spiffe/spiffe/blob/master/standards/SPIFFE-ID.md#21-trust-domain)
-  string trust_domain = 2;
-
-  // The trust domain aliases represent the aliases of `trust_domain`.
-  // For example, if we have
-  // ```yaml
-  // trustDomain: td1
-  // trustDomainAliases: [“td2”, "td3"]
-  // ```
-  // Any service with the identity `td1/ns/foo/sa/a-service-account`,
-  // `td2/ns/foo/sa/a-service-account`, or `td3/ns/foo/sa/a-service-account`
-  // will be treated the same in the Istio mesh.
-  repeated string trust_domain_aliases = 3;
+  // The `trust_domains` field contains a list of trust domains. A request from
+  // `serviceA` to `serviceB` is accepted at `serviceB` if `serviceA` carries a
+  // valid x509 cerficicate with the trust domain from the trust_domains list.
+  repeated string trust_domains = 2;
```

Comment:
Can you document if this list can be empty, and what is the expected behavior in that case?

Reply:
Added a comment about the usage. PTAL.